docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -72,7 +72,7 @@ $ tctl sso configure github \
 > github.yaml
 ```
 <Admonition type="tip">
-  GitHub organizations and teams should be referred to by their slug, not display name.
+  GitHub teams should be referred to by their slug, not display name.
```

Comment:
It looks like organization names are not changed to lowercase, but spaces and special characters are still replaced. Check https://github.com/Nic-Test-Organiza-tion, the real name has spaces and a `~`. I'm not really sure if this only applies to the URL though, or the SSO response as well

Reply:
updated to reflect that observation